docs/studios/unity/tutorials/github_actions.md

```diff
@@ -0,0 +1,309 @@
+                .QueueInstantBot((long) botId, (botInstance) =>
+                {
+                    RGBotServerListener.GetInstance().AddClientConnectionForBotInstance(botInstance.id);
+                }, () =>
+                {
+                    Debug.LogError($"Error starting bot with ID {botId}");
+                });
+        }));
+        RGBotServerListener.GetInstance().SpawnBots();
+        
+        // Wait until at least one bot is connected. Fail the test if the connection takes too long
+        var startTime = DateTime.Now;
+        while (!RGBotServerListener.GetInstance().HasBotsRunning())
+        {
+            var timePassed = DateTime.Now.Subtract(startTime).TotalSeconds;
+            if (timePassed > TIMEOUT_IN_SECONDS) Assert.Fail($"Bots failed to connect within {TIMEOUT_IN_SECONDS} seconds");
+            yield return null;
+        }
+        
+        // Now run until all bots complete their tasks
+        while (RGBotServerListener.GetInstance().HasBotsRunning())
+            yield return null;
+        
+        // Cleanup when done
+        RGBotServerListener.GetInstance()?.StopGame();
+    }
+    
+}
+
+```
+
+At a high level, this test is doing the following:
+
+1. Loading the scene, and waiting for it to start up before activating bots
+2. Queuing up a task that will start a bot and spawn it into the scene
+3. The test then waits for at least one bot to connects. If a bot does not connect within one minute, the test will fail.
+4. Once at least one bot is connected, the test runs until all bots complete their tasks (i.e the bot calls `rg.complete()`)
+
+#### 4. Test it out with an existing bot
+
+Our test is now setup! Before we commit and push our changes, let's test it out locally with an existing bot. To do this, you can
+follow the [first bot guide](/studios/unity/tutorials/first_tutorial) or [creating bots reference](/studios/unity/unity-sdk/creating-bots/configuration)
+to create and configure a bot. As a quick test, you can make a bot that simply waits for a few seconds before completing it's test. Note
+that **having your bot call `rg.complete()` is extremely important! Without this line of code, your bot will run forever!**
+
+```javascript
+export function configureBot(rg) {
+  // No config for now
+}
+
+var startTime = null
+
+export async function processTick(rg) {
+
+  if (!startTime) startTime = new Date().getTime();
+
+  if (new Date().getTime() - startTime > 5000) {
+    // Now try to finish
+    rg.complete();
+  }
+
+}
+```
+
+## Viewing Test Results
+
+Once a test is done running, you can see your most recent test results within the Regression Games website, within the **History** tab
+of the [Running Bots](https://play.regression.gg/running-bots) page. Within this page, you can download the logs for the run, as well
+as the replay file. Follow the instructions at [here](/studios/unity/unity-sdk/in-editor-replay) for viewing this replay in your local
```

Comment:
Not a comment to resolve in this PR, but I feel like we should now fill in the validation results column on the UI with # of validations passed and failed... colored accordingly based on 100% pass or not.